tests/e2e/cucumber/features/app-provider/officeSuites.feature

```diff
@@ -19,10 +19,18 @@ Feature: Integrate with online office suites like Collabora and OnlyOffice
     And "Alice" creates a public link of following resource using the sidebar panel
       | resource         | role     | password |
       | OpenDocument.odt | Can edit | %public% |
-    And "Alice" logs out
     And "Anonymous" opens the public link "Link"
     And "Anonymous" unlocks the public link with password "%public%"
-    Then "Anonymous" should see the content "OpenDocument Content" in editor "Collabora"
```

Comment:
is this step deleted on purpose? or will that not work? better to put it  as well if it works.